data.sql

```diff
@@ -13,7 +13,7 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Blossom', '1998-10-13', 3, TRUE, 17.0);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Ditto', '2022-03-14', 4, TRUE, 22.0);
 
--- Data from Project #3
+-- Owners data
```

Comment:
Hi @SirriRyisa,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![You can do it](https://media.giphy.com/media/FemTGpEZeK27M5fKL4/giphy.gif)

 ## Required Changes ♻️

- [ ] As the previous reviewer mentioned, you needed to add the missing 2 files, but those files need to have the required content. 

## Explanation

Instead of adding the required content, you only pushed 2 empty files.  We need to complete the project by adding the required content to those files.

## Hints

 - In schema.sql, you need to create the table, according to this:
 
![image](https://user-images.githubusercontent.com/57421823/163294659-0a242b7f-68f4-4d19-b929-c2ac22483477.png)

 - In queries.sql, you need to create the required queries, according to this:

![image](https://user-images.githubusercontent.com/57421823/163294700-26bdc002-8da3-4260-85c0-efc00ce82300.png)

Please, refer to the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md), and follow the instructions closely using the readings and lessons previous to this project.

That way, you will learn to create a table, and will practice creating basic queries 💪